.github/scripts/cleanup_cosi_resources.sh

```diff
@@ -41,6 +41,9 @@ log_and_run echo "Deleting Bucket Class and Bucket Claim..."
 log_and_run kubectl delete -f cosi-examples/bucketclass.yaml || { echo "Bucket Class not found." | tee -a "$LOG_FILE"; }
 log_and_run kubectl delete -f cosi-examples/bucketclaim.yaml || { echo "Bucket Claim not found." | tee -a "$LOG_FILE"; }
 
+log_and_run echo "Deleting s3-secret-for-cosi secret..."
+log_and_run kubectl delete secret s3-secret-for-cosi --namespace=default || { echo "Secret s3-secret-for-cosi not found." | tee -a "$LOG_FILE"; }
```

Comment:
Not sure if it makes any difference, but in other `kubectl` commands the `scality-object-storage` namespace is being used.

Reply:
the secret is intentionally being deployed in default namespace test the rbac and COSi driver across namespaces as cosi driver reads the secrets.